runner/internal/metrics/metrics.go

```diff
@@ -0,0 +1,43 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/benchttp/engine/runner/internal/recorder"
+)
+
+// Aggregate is an aggregate of metrics resulting from
+// from recorded requests.
+type Aggregate struct {
+	Min, Max, Avg                          time.Duration
+	SuccessCount, FailureCount, TotalCount int
+	// Median, StdDev            time.Duration
+	// Deciles                   map[int]float64
+	// StatusCodeDistribution    map[string]int
+	// RequestEventsDistribution map[recorder.Event]int
```

Comment:
Is there a particular reason these are commented out?

Reply:
Yep: not implemented!